unitTests/ui/QuerySuggestPreviewTest.ts

```diff
@@ -68,9 +72,34 @@ export function QuerySuggestPreviewTest() {
       suggestionManager.updateSuggestions(suggestions);
     }
 
+    function buildQuerySuggestPreviewAnalyticsMock(): IOmniboxAnalytics {
```

Comment:
Rather than copy-pasting the mock logic, it would be better to store the logic in a function and import it into the different files. When `IOmniboxAnalytics` changes, we will only need to update one function.